styles/typography.less

```diff
@@ -18,3 +18,40 @@ a {
     color: @primary;
   }
 }
+
+/* Font-size for pages' h1 header; overwriting Bootstrap */
+
+div.page-header h1 {
+  font-size: @page-header-size;
+}
+
+@media (min-width: 992px) {
+  div.page-header h1 {
+    font-size: @page-header-size + @page-header-size/10;
+  }
+}
+
+/* Font-size for buttons; overwriting Bootstrap */
+a.btn {
```

Comment:
If you need to overwrite Bootstrap styles and you've only got a single class name, you can double it up like this: `.btn.btn`. This way you can avoid restricting the style change to `a`. In many cases, for example, we may use `button` elements for buttons.

Reply:
Didn't know that. Thanks!

Reply:
On second thought, it's probably better to put themes into `/components/buttons.less`.

I realize now I haven't give you an introduction to the structure of style files. Let me know if you want to jump on a quick call to walk though that.

Reply:
Done.